### ISSUE TITLE
Возможна ли работа с ajax/fetch? 

### ISSUE THREAD

**Issue by IIIu6ko** [AddMoreScripts/hystModal#22]
Добрый день. При посадке на CMS возникла проблема с вашей модалкой. Алгоритм такой: отправляю запрос на сервер, обратно получаю html модалки, вставляю его на страницу и открываю.
Всё срабатывает, но модалка потом не закрывает. Метода для обновления или реинита(init() я так понял нужен только для первой инициализации, как обновление он не работает) я не нашёл. Причем не работает только закрытие, открывается подгруженная модалка вполне корректно. Пришлось подгружать внутренности модалки, а саму структуру не трогать.

Планируете ли вы делать метод update или как-то пофиксить закрытие после подгрузки или может вообще как-то отдельный функционал для подобных случаев?

**Comment by AddMoreScripts** [comment]
@IIIu6ko 
Да, возможно нужно бы сделать такой метод. Обычно, я подгружаю именно внутрь модалки что-то, и потом её открываю, или открываю, сначала показывая лоадер вместо содержимого, а потом заменяя внутренний контент.
Но, возможно имеет смысл подумать, как сделать какое-то обновление слушателей событий.
Спасибо за идею.
